spec_helper.rb

```diff
@@ -36,3 +36,17 @@ def report_on_exception=(value)
   ARGV.unshift $0
   MSpecRun.main
 end
+
+def expect_syntax_error(ruby_src)
+  -> { eval(ruby_src) }.should raise_error(SyntaxError)
+end
+
+# Evaluates the given Ruby source in a temporary Module, to prevent
+# the surrounding context from being polluted with the new methods.
+def sandboxed_eval(ruby_src)
```

Comment:
This seems pretty expensive.
Since in ruby/spec every file has its own Object I think we don't need to worry about this can just define on the `self`.